src/accapi/structs.py

```diff
@@ -171,7 +171,8 @@ def __init__(self, *args):
         self.lastName = args.pop(0)
         self.shortName = args.pop(0)
         self.category = DRIVER_CATEGORY[args.pop(0)]
-        self.nationality = NATIONALITY[args.pop(0)]
+        nationality = args.pop(0)
+        self.nationality = NATIONALITY.get(nationality, f"Unknown ({nationality})")
```

Comment:
I wish there would be a way to give a default without having to duplicate that code but it's so minor that it might not be worth implementing a solution.

Reply:
I though about using `defaultdict` but then the fallback value would be a constant. Otherwise we could implement a `get_nationality_with_fallback()` method in the `enum` module or something

Reply:
I have an idea of a `defaultdict` default that uses `inspect` to get the call stack and extract the key from there, which then executes a given method and passes it the requested key that was missing.

Reply:
Well at that point we could also just implement a class `FallbackDict` which overwrites `__getitem__` or `__missing__` 😄

Reply:
I think it would require less code to do it with `traceback` haha
```
def get_defaultdict_key():
    key = <some.path[1].to.stack>
NATIONALITY = defaultdict(lambda: f"Unknown ({get_defaultdict_key()})")
```

Reply:
Sadly there's no inspection possible on built-in functions, so the only good solution is indeed `FallbackDict.__missing__`.

Reply:
Can you add something similar to this in your PR?
```
In [9]: class defaultdictkey(defaultdict):
   ...:     def __missing__(self, k):
   ...:         if self.default_factory:
   ...:             super().__setitem__(k, self.default_factory(k))
   ...:             return self[k]
   ...:         return super().__missing__(k)
   ...: d = defaultdictkey(lambda x: f"Unknown ({x})")

In [10]: d[79]
Out[10]: 'Unknown (79)'
```

Reply:
Done. Does that look good? Might move the class to a separate module

Reply:
Looking good, I would add it to the other ones as well since they are susceptible to the same issue.

Reply:
Just realised the structs.py file has not been reverted since the real fix is now in enums.py.